core/controlplane/config/templates/cloud-config-controller

```diff
@@ -348,9 +348,9 @@ coreos:
         {{ else }}--cluster-dns={{.DNSServiceIP}} \
         {{ end }}--cluster-domain=cluster.local \
         --cloud-provider=aws \
-        {{if .Experimental.Admission.Priority.Enabled}}
-        --feature-gates=PodPriority=true \
-        {{end}}\
+        {{if or (.Experimental.Admission.Priority.Enabled) (.Experimental.Admission.PersistentVolumeClaimResize.Enabled) -}}
+        --feature-gates=PodPriority={{.Experimental.Admission.Priority.Enabled}},ExpandPersistentVolumes={{.Experimental.Admission.PersistentVolumeClaimResize.Enabled}} \
+        {{end -}}\
```

Comment:
Good try! But I guess the trailing slash still breaks the commmand. Let'ts fix it if this turns out to remain problematic.

Reply:
Hi @mumoshu 
I thought so as well but it seems to be working/not showing the same problem as #1255 
Whenever i toggle either of `Priority` or `PersistentVolumeClaimResize`, the resulting kubelet service seems to pass ok.
e.g. When I enable `Priority`, the following is shown for `kubelet.service` and Controllers seems to come up.

```
--cloud-provider=aws \
--feature-gates=PodPriority=true,ExpandPersistentVolumes=false \
\
$KUBELET_OPTS
Restart=always
RestartSec=10```

As you can see, no spaces/lines that was causing the issue in #1255 as far as I can tell.